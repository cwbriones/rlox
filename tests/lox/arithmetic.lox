print 1 + 1; // expect: 2
print 2.1 * 4; // expect: 8.4
print ((1 + 2) * 3) - (6 / 3); // expect: 7
