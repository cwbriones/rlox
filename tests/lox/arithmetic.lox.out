2
8.4
7
